openedx/core/djangoapps/programs/tests/test_utils.py

```diff
@@ -741,3 +745,15 @@ def test_course_enrollment_status(self, start_offset, end_offset, is_enrollment_
         data = utils.supplement_program_data(self.program, self.user)
 
         self._assert_supplemented(data, is_enrollment_open=is_enrollment_open)
+
+    @ddt.data(
+        timezone.now() - datetime.timedelta(days=1),
+        timezone.now(),
+        timezone.now() + datetime.timedelta(days=1)
+    )
+    def test_course_course_ended(self, course_end):
+        self.course.end = course_end
+        self.course = self.update_course(self.course, self.user.id)  # pylint: disable=no-member
+        data = utils.supplement_program_data(self.program, self.user)
+
+        self._assert_supplemented(data, is_enrollment_open=False)
```

Comment:
No enrollment start date has been specified in this test, so enrollment should be open.